src/install/index.ts

```diff
@@ -263,11 +263,7 @@ async function symlinkSelf (target: string, pkg: Package, depth: number) {
   await mkdirp(path.join(target, 'node_modules'))
   await relSymlink(
     path.join('..', '_'),
```

Comment:
We have to check if `pkg.name` contains `/` (scoped package) and therefore change `..` to `../..`.